### DISCUSSION TITLE
Multiple devices, same WebDAV

### DISCUSSION THREAD

**Discussion by base2taiji** [seedvault-app/seedvault#658] [Q&A] [answered]
I've setup a self-hosted WebDAV with Dynamic DNS and TLS for use with Seedvault backups. I wanted to do the same for my wife's phone, but I'm not sure if the second backup will mangle my phone's backup. My primary concern is Aegis authenticator backups, so alternative solutions are welcome.

**Comment by grote** [comment, marked as answer]
It should be fine to store both in the same webdav account, however, if you are worried about collisions, you could just add something like `/phone1` and `/phone2` to the webdav URL you are providing.

**Reply by base2taiji** [reply]
The solution to use different directories on the client is so obvious, clearly I've been awake for far too long haha. Anyway, thank you for your kind assistance.
